trackers/class-wp-sentry-js-tracker.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+require_once __DIR__ . '/class-wp-sentry-tracker-base.php';
+
+/**
+ * WordPress Sentry Javascript Tracker.
+ */
+final class WP_Sentry_Js_Tracker extends WP_Sentry_Tracker_Base {
+
+	/**
+	 * Get the sentry tracker instance.
+	 *
+	 * @return WP_Sentry_Js_Tracker
+	 */
+	public static function get_instance() {
+		static $instance = null;
+		return $instance ?: $instance = new self( WP_SENTRY_PUBLIC_DSN );
```

Comment:
I would rather have this be set to a static property of the class than like this, this way the `get_instance` could also be moved to the `WP_Sentry_Tracker_Base`.